db/connectDb.py

```diff
@@ -0,0 +1,60 @@
+from main import POPULATION_THRESHHOLD, CityInfo
+from dotenv import dotenv_values
+import psycopg2.extras
+
+
+config = dotenv_values('.env')
+POSTGRESQL_PASSWORD = config.get('POSTGRESQL_PASSWORD')
+POSTGRES_HOST = config.get('POSTGRES_HOST')
+POSTGRESQL_USERNAME = config.get('POSTGRESQL_USERNAME')
+
+CONN = psycopg2.connect(
+    host=POSTGRES_HOST,
+    database='postgres',
+    user=POSTGRESQL_USERNAME,
+    password=POSTGRESQL_PASSWORD
+)
+
+def getCitiesByPopulation(populationThreshold):
+    cur = CONN.cursor(cursor_factory=psycopg2.extras.DictCursor)
+
+    cur.execute(''' 
+                    SELECT city, country, lng, lat, city_id FROM worldwidecities
+                    WHERE population >= %s
+                    ORDER BY random() LIMIT 1;  
+                ''', [populationThreshold])
+
+    city = cur.fetchall()
+    for col in city:
+        city, country, lng, lat, cityId = col['city'], col['country'], col['lng'], col['lat'], col['city_id']
+    
+    cityInfo = CityInfo(city, (lng, lat), country, cityId)
+        
+    CONN.commit()
+    CONN.close()
+
+    return cityInfo
+
+def checkAnswer(cityId): 
```

Comment:
is it checking the answer or getting the answer?

Since it returns the answer(country) i would rename it `getAnswerById`